src/modules/components/vote/voteProposalDataListItem/voteProposalDataListItemStructure/voteProposalDataListItemStructure.tsx

```diff
@@ -33,22 +33,24 @@ export const VoteProposalDataListItemStructure: React.FC<IVoteProposalDataListIt
 
     return (
         <DataList.Item
-            className={classNames('flex flex-col gap-x-3 gap-y-1 md:gap-x-4 md:gap-y-1.5 md:text-lg', className)}
+            className={classNames('flex h-16 items-center gap-x-3 md:h-20 md:gap-x-4 md:text-lg', className)}
```

Comment:
Same here, also can't we directly apply the classnames required here instead of adding another wrapping div?